lib/athena/index.js

```diff
@@ -3,12 +3,14 @@
 const aws = require('aws-sdk');
 const Promise = require('bluebird');
 const _ = require('lodash');
+const clientConfig = require('../client-config');
 
 class AthenaQueryClient {
   constructor({ database, s3Bucket }) {
     this.database = database;
     this.s3Bucket = s3Bucket;
-    this.athena = new aws.Athena({});
+    const config = _.merge(_.cloneDeep(clientConfig), {});
```

Comment:
i think `const config = _.cloneDeep(clientConfig);` should be good - no need to merge with `{}`

sidenote: kind of wish all of these wrappers were a little more uniform in their initialization. would be nice to be able to pass custom config to _just_ the athena client, instead of having to rely on `clientConfig`. we could add that as a second parameter for `constructor`, but even then, it's a little weird since that's not how the other wrappers are.

Reply:
I merged with {} as the other wrappers did that. But, seems like it's not necessary since we are not providing a custom config for athena client and can be removed.